mne/epochs.py

```diff
@@ -85,7 +85,7 @@ def _save_split(epochs, fname, part_idx, n_parts):
 
     # write events out after getting data to ensure bad events are dropped
     data = epochs.get_data()
-    assert data.dtype == 'float64'
+    assert data.dtype in ['float64', 'complex64']
```

Comment:
can you us a tuple 

`('float64', 'complex64')`

also may I ask you to add / update a tiny test in test_epochs.py ?